docs/dsp-tools-create.md

```diff
@@ -1185,8 +1185,12 @@ Finally, here is a complete example of an ontology definition:
       {
         "name": "orgtpye",
         "labels": {
-          "de": "Roganisationsart",
-          "en": "Organization Type"
+          "en": "Organization Type",
+          "de": "Organisationsart",
```

Comment:
is a tailing comma valid in json? I'm not sure, honestly

Reply:
Thanks, no, it's not valid! Your comment has led me to actually try out and upload this example ontology. By doing so I found another error. But now it went through and the ontology is actually working.